src/main/resources/assets/sob/lang/zh_cn.json

```diff
@@ -0,0 +1,85 @@
+{
+  "itemGroup.sob": "点滴极乐",
+
+  "item.sob.golden_potato": "金马铃薯",
+  "item.sob.dripleaves": "垂滴叶",
+  "item.sob.nopal": "仙人掌板",
+  "item.sob.prickly_pear": "仙人球",
+  "item.sob.sculk_tendril": "幽匿脉络",
+  "item.sob.gravy": "肉汁蘸碟",
+  "item.sob.sculk_roll": "幽匿寿司",
+  "item.sob.ink_soup": "墨痕汤",
+  "item.sob.spiky_salad": "尖刺沙拉",
+  "item.sob.cinder_bread": "余烬面包",
+  "item.sob.cinder_dough": "余烬面团",
+  "item.sob.exquisite_mushroom_soup": "精致蘑菇汤",
+  "item.sob.truffle_pasta": "松露意面",
+  "item.sob.poutine": "肉汁奶酪薯条",
+  "item.sob.mangrove_stir_fry": "红树林炒菜",
+  "item.sob.veggie_wrap": "蔬菜卷饼",
+  "item.sob.duck_sandwich": "鸭肉三明治",
+  "item.sob.creeper_drink": "爆撞脏调",
```

Comment:
Questionable translation. It should not be translated as “Explosive Crash Dirty Cocktails”